data/data/azure/main.tf

```diff
@@ -171,8 +171,8 @@ resource "azurerm_storage_blob" "rhcos_image" {
   resource_group_name    = azurerm_resource_group.main.name
   storage_account_name   = azurerm_storage_account.cluster.name
   storage_container_name = azurerm_storage_container.vhd.name
-  type                   = "block"
-  source_uri             = var.azure_image_url
+  type                   = "page"
```

Comment:
any idea why this is needed here, but hasn't been necessary for RHCOS so far?
/cc @abhinavdahiya

Reply:
@jomeier could you submit a PR with just this change to master? Let's find out if changing this breaks anything in OCP.

Reply:
> @jomeier could you submit a PR with just this change to master? Let's find out if changing this breaks anything in OCP.

Ok